java/code/src/com/redhat/rhn/domain/audit/ScapFactory.java

```diff
@@ -86,7 +86,7 @@ public static void clearTestResult(long serverId, long actionId) {
                 .setLong("serverId", serverId)
                 .setLong("actionId", actionId)
                 .list();
-        results.forEach(tr -> delete(tr));
```

Comment:
im not really sure about this replacement. I'm explicitly using the other syntax because its most of the time shorter, easier to understand and also more uniform (just having a parameter and calling some function on it vs having to find a different class name every time and using that with a syntax you won't find anywhere else in the language).

Reply:
Then that could be something to discuss for a coding style. At least this is an issue for both IntelliJ and SonarQube. If we decide to not use that notation, we need to adapt both rules.

Reply:
I personally prefer `::` syntax in most cases (I let IntelliJ do the conversion for me) simply because it looks neat to me. However, I don't mind seeing the other style as well, so IMO this one should be left to the author's taste. We can happily have both in there.

Reply:
So we agreed to keep this PR as is, but not enforce this rule in the future. I changed the SonarQube profile to disable that rule already and IntelliJ users will have to change the severity of the rule in their config.

 In IntelliJ settings, go to *Editor > Inspections*, click the rule *Lambda can be replaced with method reference* and change its severity to *no highlighting, only fix*